tilt.md

```diff
@@ -0,0 +1,82 @@
+# Rapid iterative development with Tilt
+
+[Tilt](https://tilt.dev) is a tool that enables rapid iterative development of containerized workloads.
+
+Here is an example workflow without Tilt for modifying some source code and testing those changes in a cluster:
+
+1. Modify the source code.
+2. Build the container image.
+3. Either push the image to a registry or load it into your kind cluster.
+4. Deploy all the appropriate Kubernetes manifests for your application.
+    1. Or, if this is an update, you'd instead scale the Deployment to 0 replicas, scale back to 1, and wait for the
+       new pod to be running.
+
+This process can take minutes, depending on how long each step takes.
+
+Here is the same workflow with Tilt:
+
+1. Run `tilt up`
+2. Modify the source code
+3. Wait for Tilt to update the container with your changes
+
+This ends up taking a fraction of the time, sometimes on the order of a few seconds!
+
+## Installing Tilt
+
+Follow Tilt's [instructions](https://docs.tilt.dev/install.html) for installation.
+
+## Installing rukpak and catalogd
+
+operator-controller requires [rukpak](https://github.com/operator-framework/rukpak) and 
+[catalogd](https://github.com/operator-framework/catalogd). Please make sure they're installed, either normally or via 
+their own Tiltfiles, before proceeding. If you want to use Tilt, make sure you specify a unique `--port` flag to each
+`tilt up` invocation.
+
+## Starting Tilt
+
+This is typically a short as:
+
+```shell
+tilt up
+```
+
+**NOTE:** if you are using Podman, at least as of v4.5.1, you need to do this:
+
+```shell
+DOCKER_BUILDKIT=0 tilt up
+```
+
+Otherwise, you'll see an error when Tilt tries to build your image that looks similar to:
+
+```shell
+Build Failed: ImageBuild: stat /var/tmp/libpod_builder2384046170/build/Dockerfile: no such file or directory
+```
+
+When Tilt starts, you'll see something like this in your terminal:
+
+```shell
+Tilt started on http://localhost:10350/
+v0.33.1, built 2023-06-28
+
+(space) to open the browser
+(s) to stream logs (--stream=true)
+(t) to open legacy terminal mode (--legacy=true)
```

Comment:
Slightly off-topic... my workflow is `tmux` based; I don't use the graphical login session on my Linux box, so I appreciate the `(t)` option for terminal-based UI. But I also appreciate that it's simply browser-based, and _could_ use my browser from my Mac. That being said, other than the URL listed above (modified for my Linux box), are there any additional options required to allow non-localhost browsers to connect?

Reply:
I don't know, sorry

Reply:
Looks to be `--host string` -> `--host 0.0.0`
```
Host for the Tilt HTTP server and default host for any port-forwards. Set to 0.0.0.0 to listen on all interfaces. Overrides TILT_HOST env variable. (default "localhost")
```

Reply:
Cool, glad you could find it